ingestion/src/metadata/ingestion/source/database/athena/metadata.py

```diff
@@ -169,56 +172,12 @@ def create(cls, config_dict, metadata_config: OpenMetadataConnection):
             )
         return cls(config, metadata_config)
 
-    def get_tables_name_and_type(self) -> Optional[Iterable[Tuple[str, str]]]:
-        """
-        Handle table and views.
-
-        Fetches them up using the context information and
-        the inspector set when preparing the db.
-
-        :return: tables or views, depending on config
-        """
-        schema_name = self.context.database_schema.name.__root__
-        if self.source_config.includeTables:
-            for table_name in self.inspector.get_table_names(schema_name):
-                table_fqn = fqn.build(
-                    self.metadata,
-                    entity_type=Table,
-                    service_name=self.context.database_service.name.__root__,
-                    database_name=self.context.database.name.__root__,
-                    schema_name=self.context.database_schema.name.__root__,
-                    table_name=table_name,
-                )
-                if filter_by_table(
-                    self.source_config.tableFilterPattern,
-                    table_fqn if self.source_config.useFqnForFiltering else table_name,
-                ):
-                    self.status.filter(
-                        table_fqn,
-                        "Table Filtered Out",
-                    )
-                    continue
-
-                yield table_name, TableType.External  # All tables in Athena are external
-
-        if self.source_config.includeViews:
-            for view_name in self.inspector.get_view_names(schema_name):
-                view_fqn = fqn.build(
-                    self.metadata,
-                    entity_type=Table,
-                    service_name=self.context.database_service.name.__root__,
-                    database_name=self.context.database.name.__root__,
-                    schema_name=self.context.database_schema.name.__root__,
-                    table_name=view_name,
-                )
-                if filter_by_table(
-                    self.source_config.tableFilterPattern,
-                    view_fqn if self.source_config.useFqnForFiltering else view_name,
-                ):
-                    self.status.filter(
-                        view_fqn,
-                        "Table Filtered Out",
-                    )
-                    continue
-
-                yield view_name, TableType.View
+    def query_table_names_and_types(
+        self, schema_name: str
+    ) -> Iterable[TableNameAndType]:
+        """Return tables as external"""
+
+        return [
+            TableNameAndType(name=name, type_=TableType.External)
```

Comment:
no need to reimplement the full filtering logic in athena. Just map the required type when we get the inspector results